app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuAccessibilityRobot.kt

```diff
@@ -49,13 +48,7 @@ class SettingsSubMenuAccessibilityRobot {
         const val TEXT_SIZE = 16f
     }
 
-    fun verifyAutomaticFontSizingMenuItems() = assertAutomaticFontSizingMenuItems()
-
-    fun clickFontSizingSwitch() = toggleFontSizingSwitch()
-
-    fun verifyNewMenuItems() = assertNewMenuItems()
-
-    fun verifyNewMenuItemsAreGone() = assertNewMenuItemsAreGone()
+    fun verifyMenuItems() = assertNewMenuItems()
```

Comment:
[optional][nit] Should we rename assertNewMenuItems() to assertMenuItems() as well?